### ISSUE TITLE
Add `logbt --setup` method

### ISSUE THREAD

**Issue by springmeyer** [mapbox/logbt#17]
Currently logbt can be:

 - Run as root (and will set up the `core_pattern` correctly and transparently)
 - Run as non-root (and then expects the `core_pattern` to be set correctly beforehand)

The manual setting of the `core_pattern` is tedious, in the non-root case, is tedious. We can avoid this by adding a `logbt --setup` method that does it automatically. Then if you wanted to run `logbt` as non-root you'd:

```bash
# run as root just for setup
sudo logbt --setup
# then run as non-root when running your process
logbt node index.js
```

This would replace the current recommendation of:

```bash
sudo bash -c "echo '/tmp/logbt-coredumps/core.%p.%E' > /proc/sys/kernel/core_pattern"
logbt node index.js
```

**Comment by springmeyer** [comment]
Working on this at https://github.com/mapbox/logbt/compare/refactor-2.0

**Comment by springmeyer** [comment]
Landed in #22. Will be updating the docs and releasing 2.0 shortly.
